examples/resources/awscc_chatbot_slack_channel_configuration/first_example.tf

```diff
@@ -0,0 +1,14 @@
+
+resource "awscc_chatbot_slack_channel_configuration" "example" {
+  # Set the name to identify the configuration in the ChatBot console
```

Comment:
you don't have to add the comment inside the resources, the schema explain what are each attributes are for

Reply:
Will fix